### DISCUSSION TITLE
[4.0.0] Custom SSL certificates

### DISCUSSION THREAD

**Discussion by andrasbacsai** [coollabsio/coolify#3182] [Feature Requests]
Able to add custom SSL certificates to Traefik.

**Comment by AlejandroAkbal** [comment]
Yes please

**Comment by swissbyte** [comment]
Check this out #1483 could be added as featurer request there…

**Comment by mrroll** [comment]
I think this could be useful for instances where folks want to be behind Cloudflare and use a self signed certificate.

**Comment by alan910127** [comment]
Hello, I just noticed this issue and would like to see it implemented. Is there any workaround available right now?

**Comment by peaklabs-dev** [comment]
When is this coming, I want to use Cloudflare origin SSL certificates on each of my services?

**Comment by ladderschool** [comment]
I thought this is already implemented through: https://coolify.io/docs/knowledge-base/traefik/custom-ssl-certs

**Comment by AlejandroAkbal** [comment]
Yeah, it’s now implemented, could be closed

**Comment by peaklabs-dev** [comment]
No this is for adding SSL certs via the UI not trough ssh and folders.

**Comment by ladderschool** [comment]
> No this is for adding SSL certs via the UI not trough ssh and folders.

Oh I see! Yes this would be fantastic. I could sign up with any provider like namecheap and download the key/certs and upload it through the Coolify UI you're saying? That would be amazing.
